frontend/src/components/dashboard/dashboard.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react/addons';
+import {Navigation} from 'react-router';
+import Project from './project.jsx';
+import {DashboardActions as Actions} from '../../actions/actions.js';
+import dashboardStore from '../../stores/dashboardStore.js';
+
+let Dashboard = React.createClass({
+  mixins: [Navigation],
+
+  getInitialState() {
+    return {
+      projects: []
+    };
+  },
+
+  // listen to data changes from `dashboardStore`
+  // `listen` returns a convenient unsubscribe functor
+  componentDidMount() {
+    this.unsubscribe = dashboardStore.listen(this.onFetchProjects);
+    Actions.fetchProjects();
+  },
+
+  componentWillUnmount() {
```

Comment:
Is there any benefit to using this compared to the ListenerMixin?
https://github.com/reflux/refluxjs#convenience-mixin-for-react